session_4/blog/app/controllers/users_controller.rb

```diff
@@ -0,0 +1,75 @@
+class UsersController < ApplicationController
+  load_and_authorize_resource
+  before_action :set_user, only: [:show, :edit, :update, :destroy]
+
+  # GET /users
+  # GET /users.json
+  def index
+    @users = User.all
+  end
+
+  # GET /users/1
+  # GET /users/1.json
+  def show
```

Comment:
If you want, you can show the articles written by a user when you go to their profile page.

Reply:
Thank you for your suggestions. I have implemented some of them.
[https://github.com/hrushikeshj/IRIS-RoR-Bootcamp-2020/tree/session_4_hrushikesh/session_4#user-profile-page](https://github.com/hrushikeshj/IRIS-RoR-Bootcamp-2020/tree/session_4_hrushikesh/session_4#user-profile-page)